bin/inference/pycbc_inference_plot_posterior

```diff
@@ -54,6 +56,12 @@ parser.add_argument("--output-file", type=str, required=True,
                     help="Output plot path.")
 parser.add_argument("--verbose", action="store_true", default=False,
                     help="Be verbose")
+parser.add_argument("--plot-prior", nargs="+", type=str,
```

Comment:
The name isn't making it clear that it's for inputting a config file. Something like `--prior-config-file` could make it clear.

Reply:
It says in the help message that you need to provide one or more config file(s) for this option.